# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
#
# Peter Lyberth <peterjensen84@gmail.com>, 2009.
msgid ""
msgstr ""
"Project-Id-Version: da\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2009-11-23 12:19+0200\n"
"PO-Revision-Date: 2009-09-08 12:23+0200\n"
"Last-Translator: Peter Lyberth <peterjensen84@gmail.com>\n"
"Language-Team: Danish <pardus-translators@pardus.org.tr>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Lokalize 0.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: bin/mudur.py:207
msgid "Cannot write mudur.log, read-only file system"
msgstr "Kan ikke oprette murdur.log, filsystemet er læs-kun"

#: bin/mudur.py:373
msgid "Safe Mode"
msgstr "Sikker tilstand"

#: bin/mudur.py:376
msgid "Cannot find /etc/pardus-release"
msgstr "Kan ikke finde /etc/pardus-release"

#: bin/mudur.py:492
#, python-format
msgid "Could not set unicode mode on tty %d"
msgstr "Kunne ikke sætte tty %d til unicode tilstand"

#: bin/mudur.py:559
#, python-format
msgid "Bringing up %s"
msgstr ""

#: bin/mudur.py:564
#, python-format
msgid "Unable to bring up %s"
msgstr ""

#: bin/mudur.py:634
msgid "No network connection, skipping remote mount."
msgstr ""

#: bin/mudur.py:644
msgid "Cannot connect to DBus, services won't be started"
msgstr ""

#: bin/mudur.py:665
#, python-format
msgid ""
"Unable to start network:\n"
"  %s"
msgstr ""

#: bin/mudur.py:670
msgid "Cannot start system logger"
msgstr ""

#: bin/mudur.py:673
msgid "Starting services"
msgstr ""

#: bin/mudur.py:705
msgid "Stopping services"
msgstr ""

#: bin/mudur.py:725 bin/mudur.py:745 bin/service.py:158 bin/service.py:261
#, python-format
msgid "Starting %s"
msgstr "Starter %s"

#: bin/mudur.py:735 bin/mudur.py:752 bin/service.py:169 bin/service.py:196
#: bin/service.py:273
#, python-format
msgid "Stopping %s"
msgstr ""

#: bin/mudur.py:777
msgid "Can't move persistent udev rules from /dev/.udev"
msgstr ""

#: bin/mudur.py:783
msgid "Mounting /dev"
msgstr ""

#: bin/mudur.py:825
msgid "Starting udev"
msgstr ""

#: bin/mudur.py:839
msgid "Populating /dev"
msgstr ""

#: bin/mudur.py:874
msgid "Removing stale lock file /etc/mtab~"
msgstr ""

#: bin/mudur.py:877
msgid "Failed removing stale lock file /etc/mtab~"
msgstr ""

#: bin/mudur.py:888
msgid "/etc/fstab doesn't contain an entry for the root filesystem"
msgstr ""

#: bin/mudur.py:894
msgid "Remounting root filesystem read-only"
msgstr ""

#: bin/mudur.py:899
msgid "Checking root filesystem (full check forced)"
msgstr ""

#: bin/mudur.py:905
msgid "Checking root filesystem"
msgstr ""

#: bin/mudur.py:914
msgid "Filesystem repaired, but reboot needed!"
msgstr ""

#: bin/mudur.py:918
msgid "Rebooting in 10 seconds..."
msgstr ""

#: bin/mudur.py:920
msgid "Rebooting..."
msgstr ""

#: bin/mudur.py:924
msgid "Filesystem could not be repaired"
msgstr ""

#: bin/mudur.py:929
msgid "Skipping root filesystem check (fstab's passno == 0)"
msgstr ""

#: bin/mudur.py:935
msgid "Remounting root filesystem read/write"
msgstr ""

#: bin/mudur.py:939
msgid ""
"Root filesystem could not be mounted read/write\n"
"   You can either login below and manually check your filesytem(s) OR\n"
"   restart your system, press F3 and select 'FS check' from boot menu\n"
msgstr ""

#: bin/mudur.py:951
msgid "Couldn't synchronize /etc/mtab from /proc/mounts"
msgstr ""

#: bin/mudur.py:970
msgid "Checking all filesystems"
msgstr ""

#: bin/mudur.py:974
msgid "A full fsck has been forced"
msgstr ""

#: bin/mudur.py:992
msgid "Filesystem errors corrected"
msgstr ""

#: bin/mudur.py:994
msgid "Fsck could not correct all errors, manual repair needed"
msgstr ""

#: bin/mudur.py:1002
msgid "Mounting USB filesystem"
msgstr ""

#: bin/mudur.py:1005
msgid "Mounting local filesystems"
msgstr ""

#: bin/mudur.py:1025
msgid "Starting portmap service for NFS"
msgstr ""

#: bin/mudur.py:1028
msgid "Mounting remote filesystems (CTRL-C stops trying)"
msgstr ""

#: bin/mudur.py:1042
msgid "Mounting skipped with CTRL-C, remote shares will not be accessible!"
msgstr ""

#: bin/mudur.py:1085
#, python-format
msgid "Setting up hostname as '%s'"
msgstr ""

#: bin/mudur.py:1110
msgid "Setting disk parameters"
msgstr ""

#: bin/mudur.py:1132
msgid "Activating swap space"
msgstr ""

#: bin/mudur.py:1141
msgid "Deactivating swap space"
msgstr ""

#: bin/mudur.py:1150
msgid "Cleaning up /var"
msgstr ""

#: bin/mudur.py:1161
msgid "Cleaning up /tmp"
msgstr ""

#: bin/mudur.py:1197
msgid "Setting system clock to hardware clock"
msgstr ""

#: bin/mudur.py:1213
msgid "Failed to adjust systematic drift of the hardware clock"
msgstr ""

#: bin/mudur.py:1217
msgid "Failed to set system clock to hardware clock"
msgstr ""

#: bin/mudur.py:1228
msgid "Syncing system clock to hardware clock"
msgstr ""

#: bin/mudur.py:1231
msgid "Failed to synchronize clocks"
msgstr ""

#: bin/mudur.py:1258
msgid "Unmounting filesystems"
msgstr ""

#: bin/mudur.py:1290
msgid "Remounting remaining filesystems read-only"
msgstr ""

#: bin/mudur.py:1306
msgid ""
"An internal error occured. Please report to the bugs.pardus.org.tr with "
"following information:"
msgstr ""

#: bin/mudur.py:1367
msgid "Mounting /sys"
msgstr ""

#: bin/mudur.py:1381
msgid "Mounting /dev/pts"
msgstr ""

#: bin/mudur.py:1430
msgid "Setting up localhost"
msgstr ""

#: bin/mudur.py:1443
msgid "Updating environment variables"
msgstr ""

#: bin/mudur.py:1464
msgid "Triggering udev events which are failed during a previous run"
msgstr ""

#: bin/mudur.py:1503
msgid "Trying to initiate a warm reboot (skipping BIOS with kexec kernel)"
msgstr ""

#: bin/service.py:60
msgid "local"
msgstr ""

#: bin/service.py:61
msgid "script"
msgstr ""

#: bin/service.py:62
msgid "server"
msgstr ""

#: bin/service.py:74
msgid "running"
msgstr ""

#: bin/service.py:76
msgid "yes"
msgstr ""

#: bin/service.py:78
msgid "conditional"
msgstr ""

#: bin/service.py:101
msgid "Status"
msgstr ""

#: bin/service.py:102
msgid "Service"
msgstr ""

#: bin/service.py:103
msgid "Autostart"
msgstr ""

#: bin/service.py:104
msgid "Description"
msgstr ""

#: bin/service.py:181
#, python-format
msgid "Service '%s' will be auto started."
msgstr ""

#: bin/service.py:183
#, python-format
msgid "Service '%s' won't be auto started."
msgstr ""

#: bin/service.py:185
#, python-format
msgid "Service '%s' will be started if required."
msgstr ""

#: bin/service.py:250
msgid "You must be root to use that."
msgstr ""

#: bin/service.py:269
msgid "Unable to start DBus"
msgstr ""

#: bin/service.py:283
msgid "DBus is not running."
msgstr ""

#: bin/service.py:285
msgid "DBus is running."
msgstr ""

#: bin/service.py:290
msgid ""
"usage: service [<options>] [<service>] <command>\n"
"where command is:\n"
" list     Display service list\n"
" status   Display service status\n"
" info     Display service status\n"
" on       Auto start the service\n"
" off      Don't auto start the service\n"
" start    Start the service\n"
" stop     Stop the service\n"
" restart  Stop the service, then start again\n"
" reload   Reload the configuration (if service supports this)\n"
"and option is:\n"
" -N, --no-color  Don't use color in output\n"
" -q, --quiet     Don't print replies"
msgstr ""

#: bin/service.py:346
#, python-format
msgid "No such service: %s"
msgstr ""
